package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cros-dpsl-js",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
Can we sync DPSL version with Chrome OS version where new API first introduced?

Reply:
I understand the advantage of that but I don't think it is a good idea to follow something else than the community standards: https://docs.npmjs.com/about-semantic-versioning

Users of this package are always recommended to use the latest version in order to get the latest features and bug fixes.

What we can do is to maintain release notes as part of the repo, for example: https://github.com/npm/cli/releases

Reply:
ack, thanks for giving more info!